docker-compose-emulated.yml

```diff
@@ -263,8 +263,6 @@ services:
     restart: always
     image: mysql:8.3.0
     container_name: mysql
-    expose:
-      - "3306"
```

Comment:
Nowadays this is mostly just for documentation purposes (so it appears when we do ``docker inspect``), so docker compose add the tag to the container/image built during up. Useless at runtime.